### DISCUSSION TITLE
Make the first-line indentation in the summary more consistent

### DISCUSSION THREAD

**Discussion by rosystain** [gotson/komga#1653] [Ideas]
Hello, I've noticed that Komga removes the indentation or spaces at the beginning of the text in the summary area.

However, first-line indentation is a common text formatting practice in some languages such as Chinese.

It is not absolute, but the indentation character will appear before each paragraph if it uses first-line indentation formatting.

The problem is that Komga only removes the first-line indentation of the first paragraph. This makes the text formatting inconsistent, as shown in the image.

<img width="1241" alt="屏幕截图 2024-08-18 215109" src="https://github.com/user-attachments/assets/a0b7a2d9-b717-4419-9849-3ea8b0f7e971">

Perhaps Komga could consider removing the indentation from each paragraph like the image below, though retaining all first-line indentations is also welcome.

[
<img width="1262" alt="屏幕截图 2024-08-18 221647" src="https://github.com/user-attachments/assets/44a488c0-1609-4cc8-be33-ff4b941de648">
](url)

**Comment by gotson** [comment]
There's no specific handling in Komga about removing anything. Could be a default behaviour from some components or framework used for display.

Can you share exactly what character is used for indentation, in UTF8 coding ?

**Reply by rosystain** [reply]
Just using space characters for indentation.

For example, I write two paragraphs, each with 4 space characters as indentation.



<img width="602" alt="屏幕截图 2024-08-19 123603" src="https://github.com/user-attachments/assets/ca12b20c-e97f-4dc6-a447-cc2620240ed4">

Then click 'Save,'  the indentation of the first paragraph has disappeared.

<img width="503" alt="屏幕截图 2024-08-19 123617" src="https://github.com/user-attachments/assets/ec3c1635-a6f8-4436-a55b-e205bf211f2d">

When I open the editing interface again, I notice that the 4 spaces at the beginning of the first paragraph in the summary input box have also disappeared.


<img width="605" alt="屏幕截图 2024-08-19 123632" src="https://github.com/user-attachments/assets/07418291-4293-40df-a341-2bf2193788ab">

**Reply by gotson** [reply]
text data fields are trimmed, meaning extra space at the beginning/end are removed before being saved in database
